web/client/reducers/config.js

```diff
@@ -21,10 +22,15 @@ function mapConfig(state = null, action) {
         let hasVersion = action.config.version && action.config.version >= 2;
             // we get from the configuration what will be used as the initial state
         let mapState = action.legacy && !hasVersion ? ConfigUtils.convertFromLegacy(action.config) : ConfigUtils.normalizeConfig(action.config.map);
-
-        mapState.map = assign({}, mapState.map, {mapId: action.mapId, size, version: hasVersion ? action.config.version : 1});
+        const newMapState = assign({}, set(mapState, "layers", mapState.layers.map( l => {
+            if (l.group === "background" && (l.type === "ol" || l.type === "OpenLayers.Layer")) {
+                l.type = "empty";
```

Comment:
Change new.json to align everything to the type "empty"

Reply:
done

Reply:
```
const newMapState = {
    ...mapState,
   layers : ...
}
```

You are doing something like: 
```
var temp = mapState.layers = ...
{
  ...mapState: 
 layers: temp
}
```
so you are still mutating